src/rgw/rgw_auth.h

```diff
@@ -396,17 +399,18 @@ class RemoteApplier : public IdentityApplier {
   const acl_strategy_t extra_acl_strategy;
 
   const AuthInfo info;
-  const bool implicit_tenants;
+  std::tuple<bool,bool> (*implicit_tenants)(CephContext*const);
```

Comment:
I vote for a tiny `struct` with descriptively named members for the sake of readability. Though, this still is a cosmetics.